src/main/java/uk/gov/hmcts/reform/userprofileapi/util/JsonFeignResponseHelper.java

```diff
@@ -43,21 +46,25 @@ private JsonFeignResponseHelper() { }
 
     public static <T> Optional<T> decode(Response response, Optional<Class<T>> clazz) {
         Optional<T> result = Optional.empty();
-        if (isStatusCodeSuccessful(response.status()) && clazz.isPresent()) {
+        if (clazz.isPresent()) {
             try {
                 Optional<Collection<String>> encodings = Optional.ofNullable(response.headers().get("content-encoding"));
                 result = Optional.of((encodings.isPresent() && encodings.get().contains("gzip"))
                         ? json.readValue(new GZIPInputStream(new BufferedInputStream(response.body().asInputStream())), clazz.get())
                         : json.readValue(response.body().asReader(Charset.defaultCharset()), clazz.get()));
             } catch (IOException e) {
-                log.warn("Error could not decode!");
+                log.warn("Error could not decoded : " + e.getLocalizedMessage());
```

Comment:
we can use {} place holder rather than +

Reply:
done